src/xtopcom/xdata/xfull_tableblock.h

```diff
@@ -29,7 +29,11 @@ class xfulltable_statistics_resource_t : public xbase_dataunit_t<xfulltable_stat
     int32_t do_read(base::xstream_t & stream) override;
 
  public:
-    const std::string & get_statistics_data() const {return m_statistics_data;}
+    const xbyte_buffer_t get_statistics_data() const {
```

Comment:
just return m_statistics_data, and change m_statistics_data from std::string to xbyte_buffer_t